pdata/pcommon/generated_primitive_slice.go

```diff
@@ -0,0 +1,209 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by "model/internal/cmd/pdatagen/main.go". DO NOT EDIT.
+// To regenerate this file run "go run model/internal/cmd/pdatagen/main.go".
+
+package pcommon
+
+import "go.opentelemetry.io/collector/pdata/internal"
+
+// ByteSlice represents a []byte slice.
+// The instance of ByteSlice can be assigned to multiple objects since it's immutable.
+//
+// Must use NewByteSlice function to create new instances.
+// Important: zero-initialized instance is not valid for use.
+type ByteSlice internal.ByteSlice
+
+// Deprecated: [0.60.0] Use ByteSlice instead.
+type ImmutableByteSlice = ByteSlice
```

Comment:
would not bother to generate the deprecated funcs/types. lets write them manually.

Reply:
done